### ISSUE TITLE
Update deprecated light and switch components

### ISSUE THREAD

**Issue by 1h8fulkat** [JoshuaMulliken/ha-wyzeapi#52]
```
2020-05-21 14:35:46 DEBUG (MainThread) [homeassistant.core] Bus:Handling <Event component_loaded[L]: component=wyzeapi>
2020-05-21 14:35:46 WARNING (MainThread) [homeassistant.components.light] Light is deprecated, modify WyzeBulb to extend LightEntity
2020-05-21 14:35:46 INFO (MainThread) [homeassistant.components.light] Setting up light.wyzeapi
2020-05-21 14:35:46 WARNING (MainThread) [homeassistant.components.switch] SwitchDevice is deprecated, modify WyzeSwitch to extend SwitchEntity
2020-05-21 14:35:46 INFO (MainThread) [homeassistant.components.switch] Setting up switch.wyzeapi
```

Home assistant .110 has been throwing the following errors relating to the HA-WYZEAPI integration.  Looks like the integration needs to update the component references.

**Comment by 1h8fulkat** [comment]
I thought this was resolved, but it's still appearing in the logs

**Comment by JoshuaMulliken** [comment]
Gotcha. Thanks for the heads up.

**Comment by JoshuaMulliken** [comment]
Fixed in commit 0c8b000f46d6aac1954e3b27a5295c90140af068
